x-pack/plugins/canvas/server/routes/workpad.test.js

```diff
@@ -5,9 +5,18 @@
  */
 
 import Hapi from 'hapi';
-import { CANVAS_TYPE, API_ROUTE_WORKPAD } from '../../common/lib/constants';
+import {
+  CANVAS_TYPE,
+  API_ROUTE_WORKPAD,
+  API_ROUTE_WORKPAD_ASSETS,
+  API_ROUTE_WORKPAD_STRUCTURES,
+} from '../../common/lib/constants';
 import { workpad } from './workpad';
 
+const routePrefix = API_ROUTE_WORKPAD;
+const routePrefixAssets = API_ROUTE_WORKPAD_ASSETS;
+const routePrefixStructures = API_ROUTE_WORKPAD_STRUCTURES;
```

Comment:
Are the routes kept separate because route unification is tangential to what this PR is needed for, or because you have second thoughts about its desirability? (not suggesting that unification be done in this one, just wishing to know esp. if it's the latter)

Reply:
> because route unification is tangential to what this PR is needed for

That ;)

Having a single handler makes the changes I'm planning easier, the route unification is separate, and still seems worth doing. And now that there's tests, we can just update the api path in them and confirm they still work they same way.